features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/services/login/token_callback.jag

```diff
@@ -0,0 +1,64 @@
+<%
+    var log = new Log("Token Request Function");
+
+    var tokenRequestData = {
+        "grant_type": "authorization_code",
+        "code": request.getParameter("code","UTF-8"),
+        "redirect_uri": "https://localhost:9443/store-new/services/auth/callback",
+    };
+
+    var SystemApplicationDTO = Packages.org.wso2.carbon.apimgt.impl.dao.SystemApplicationDAO;
+    var systemApplicationDAO = new SystemApplicationDTO();
+    var systemApplicationDTO = systemApplicationDAO.getClientCredentialsForApplication("admin_store");
+
+    var clientId = systemApplicationDTO.getConsumerKey();
+    var clientSecret = systemApplicationDTO.getConsumerSecret();
+    var Base64 = org.apache.axiom.om.util.Base64;
+    var String = Packages.java.lang.String;
+    var base64encoded = Base64.encode ( new String (clientId + ":" + clientSecret).getBytes());
+    log.debug(base64encoded);
```

Comment:
Let's remove this debug log as it is logging the secret value.

Reply:
resolved